feature_engine/discretisation/arbitrary.py

```diff
@@ -148,12 +148,16 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
         return self
 
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
-        """Sort the variable values into the intervals.
```

Comment:
could we please remove this file from the PR?

Reply:
Actually, we are now inheriting precision, so I think we need to add the docstrings instead in the init. Could you please check and see if we need to remove or update this file?